renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -1975,7 +1975,8 @@ bool GLReplay::RenderTextureInternal(TextureDisplay cfg, int flags)
     ubo->Channels.x = 1.0f;
     ubo->Channels.y = 0.0f;
     ubo->Channels.z = 0.0f;
-    ubo->Channels.w = 0.0f;
+    // don't want to splat the red channel when writing depth value, therefore making alpha non-zero
+    ubo->Channels.w = dsTexMode == eGL_DEPTH_COMPONENT ? 1.0f : 0.0f;
```

Comment:
The reason is that by splatting the depth to all 4 channels, the depth values will get copied to the stencil part even if the original format doesn't contain stencil. It's just to prevent a depth only format to have nonzero stencil value.